client/css/style.css

```diff
@@ -587,11 +587,8 @@ button {
 }
 
 #main {
-	background: #fff;
```

Comment:
In short, this is always shadowed by [https://github.com/thelounge/lounge/pull/572/files#diff-97db1f70168fb5f12457b238ff6052b5R650](https://github.com/thelounge/lounge/pull/572/files#diff-97db1f70168fb5f12457b238ff6052b5R650).